diagnostic.md

```diff
@@ -54,10 +70,10 @@ const ask = require('../lib/ask.js');
 
 let answer = 'not empty';
 
-while (answer !== '') {
-  answer = ask("Guess my secret? ");
-  if (answer === 'SeCrEt') {
-   console.log("You guessed it!");
+for (answer = '') {
+  answer = ask("Guess my number, it is between 1 and 10? ");
+  if (answer<=10) {
+     console.log("You guessed it!");
    break;
   }
 }
```

Comment:
Remember, `for` loops run on a count, while `while` loops run if a condition evaluates as true. Here, since the answer you're waiting for could come in at anytime, it makes more sense to use a `while` loop.


```
while (answer !== '') {
  answer = ask("Guess my secret? ");
  if (answer >=1 && answer <=10) {
   console.log("You guessed it!");
   break;
 }else{
   console.log('Please try a number between 1 and 10')
 }
}
```